tokio/src/time/mod.rs

```diff
@@ -99,7 +99,7 @@ pub use sleep::{sleep, sleep_until, Sleep};
 pub(crate) mod driver;
 
 mod error;
-pub use error::Error;
+pub use error::{Elapsed, Error};
```

Comment:
This import should be removed. Then we can avoid cluttering the main namespace with error types :)

Reply:
Fixed. Hopefully I understood correctly how they should be exposed.

Reply:
btw @alce I think he means just doing `pub mod error` then you can address each error variant via `error::Elapsed` etc.